frontend/shared/src/components/header/Header.tsx

```diff
@@ -63,15 +74,37 @@ const Header: React.FC<HeaderProps> = ({
           ))}
         </Navigation.Row>
       )}
-      {languages && (
-        <Navigation.Actions>
+
+      <Navigation.Actions>
+        {login && (
+          <Navigation.User
+            authenticated={!!login.userName}
```

Comment:
It's not possible because auth check is done against the /userinfo endpoint which always returns userdata. But this is shared component and perhaps a bit lazy way to check it. I can add another prop for authentication to make it more clear.